src/commands/prefix/music.ts

```diff
@@ -0,0 +1,55 @@
+import { exec } from "child_process"
+import { PrefixCommand } from "core/interfaces/command"
+import { Message } from "discord.js"
+
+
+let musicDir = '~/Documents/brock/code/saigo/data/music/'
+
+export const music: PrefixCommand = {
+    name: 'music',
+    description: 'Music command: usage !music <help|fetch|play|stop|skip> <link>',
+    run: async (message: Message, parsedMsg: Array<string>) => {
+        let subCmd = parsedMsg[1]
+        let link = parsedMsg[2]
```

Comment:
if we have a command that only has two parameters, e.g. `!music random` would this fail ? maybe should parse `link` only if we know its `fetch`